backend/src/main/java/com/htecgroup/skynest/service/EmailService.java

```diff
@@ -0,0 +1,5 @@
+package com.htecgroup.skynest.service;
+
+public interface EmailService {
+  void send(String to, String email, String subject);
```

Comment:
As I wrote earlier, you can create a new object for email metadata and sent that as input param. For every different mail, that object will contain different data, but the same logic will be applied for sending mail .